bids/layout/layout.py

```diff
@@ -129,63 +95,58 @@ class BIDSLayout(object):
         in the root argument is reindexed. If False, indexing will be
         skipped and the existing database file will be used. Ignored if
         database_path is not provided.
-    index_metadata : bool
-        If True, all metadata files are indexed at
-        initialization. If False, metadata will not be available (but
-        indexing will be faster).
+    indexer: BIDSLayoutIndexer
```

Comment:
Can we link to the object? Something like

```suggestion
    indexer: :class:`.index.BIDSLayoutIndexer`
```